src/components/TimeoutNotification/TimeoutNotification.js

```diff
@@ -0,0 +1,73 @@
+import React, {Component} from 'react';
+import {Redirect} from 'react-router';
+import './TimeoutNotification.css';
+
+const WAIT_BEFORE_DISPLAY = 10;
```

Comment:
10 seconds seems a bit short

Reply:
This is for no user interaction at all. Also it won't be on the item scan page as that has no user to screen interaction, so it should be long enough